### DISCUSSION TITLE
Issue TCP close

### DISCUSSION THREAD

**Discussion by quentho2617** [moscajs/aedes#954] [General]
Hello, 

Hello,

I am developing connected devices using MQTT over a TCP layer. I have installed your library on an EC2 server on AWS. The connection works well, but occasionally the server closes the TCP connection. Have you encountered this kind of problem before, and do you know what might be causing it?

Thank you in advance.
![image](https://github.com/moscajs/aedes/assets/39057713/8f3883fc-7347-4f06-84e4-e234a4d56a98)

**Comment by robertsLando** [comment]
This usually happens when the client doesn't send any pingreq in 1.5*keeepalive time so the broker consider the client offline and closes the connection
